src/App.js

```diff
@@ -1,25 +1,120 @@
-import logo from './logo.svg';
-import './App.css';
+import logo from "./logo.svg";
+import "./App.css";
+import {useEffect, useState} from "react";
+import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
+import {faTrashCan} from "@fortawesome/free-regular-svg-icons";
 
-function App() {
+function NavBar() {
+  return (
+    <nav
+      className="flex flex-wrap justify-center items-center gap-4 border-b-2 border-b-gray-200 px-7 py-3 font-display">
+      <a href="/">
+        <img src={logo} width="40" alt="logo"/>
+      </a>
+      <a className="text-4xl font-extrabold text-primary" href="/">tomo</a>
+      <span className="sm:flex-grow"></span>
+      <div className="block md:inline">
+        <a className="text-2xl text-neutral-gray font-bold hover:text-primary" href="/classes">Classes</a>
+        <a className="text-2xl text-neutral-gray font-bold ml-8 hover:text-primary" href="/chats">Chats</a>
+        <a className="text-2xl text-neutral-gray font-bold ml-8 hover:text-primary" href="/profile">Profile</a>
+      </div>
+    </nav>
+  );
+}
+
+function Footer() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="mt-48 px-8 py-5 border-t-2 border-t-gray-200 w-full flex flex-wrap gap-8">
+      <span className="text-neutral-gray text-sm">© 2024 Tomo</span>
+      <a href="/about" className="text-neutral-gray text-sm underline underline-offset-2">About</a>
     </div>
   );
 }
 
-export default App;
+const CLASSES = [
+  {
+    id: 0,
+    title: "COMP_SCI 211-0",
+    description: "Fundamentals of Computer Programming II",
+    members: 67,
+    url: "/classes/comp-sci-211-0"
+  },
+  {
+    id: 1,
+    title: "COMP_ENG 203-0",
+    description: "Introduction to Computer Engineering",
+    members: 48,
+    url: "/classes/comp-eng-203-1"
+  },
+  {
+    id: 2,
+    title: "MATH 228-1",
+    description: "Multivariable Differential Calculus for Engineering",
+    members: 61,
+    url: "/classes/math-228-1"
+  },
+  {
+    id: 3,
+    title: "GEN_ENG 206-1",
+    description: "Honors Engineering Analysis 1",
+    members: 56,
+    url: "/classes/gen-eng-206-1"
+  },
+  {
+    id: 4,
+    title: "CONDUCT 393-0",
+    description: "Orchestral Organizations",
+    members: 77,
+    url: "/classes/conduct-393-0"
+  }
+];
+
+function ClassCard(props) {
```

Comment:
everything is nicely organized and readable within App.js. one thing i'd suggest, especially as you expand on this initial implementation, is separating components out into their own js files (e.g. ClassCard.js). makes it easier to reuse and style separately, and is generally just good practice in React :)